libs/barista-components/breadcrumbs/src/breadcrumbs.ts

```diff
@@ -49,38 +98,394 @@ export const _DtBreadcrumbMixinBase = mixinColor<
+          this._overlayRef?.hasAttached() &&
+          firstTransplantedItem
+        ) {
+          event.preventDefault();
+          firstTransplantedItem._elementRef.nativeElement.focus();
+          return;
+        }
+
+        // Focus the next focusable item if the last item in the overlay is blurred
+        if (event.target === lastTransplantedItem?._elementRef.nativeElement) {
+          if (firstVisibleItem) {
+            event.preventDefault();
+            firstVisibleItem._elementRef.nativeElement.focus();
+          } else {
+            this._trigger?.nativeElement.focus();
+          }
+
+          return;
+        }
+      }
+    });
+  }
+
+  /** Nexts overlay trigger observable to trigger focus handling */
+  _handleOverlayTriggerKeydown($event: KeyboardEvent): void {
+    this._overlayTriggerKeydown$.next($event);
+  }
+
+  /**
+   * Creates a container in the body that holds all breadcrumb items
+   * This container is then moved into the overlay when the overlay is created
+   */
+  private _createCollapsedContainer(): void {
+    if (!this._collapsedContainer) {
+      const container = this._document.createElement('div');
+      container.id = `dt-breadcrumb-collapsed-container`;
+      container.setAttribute(`aria-expanded`, `false`);
+      container.classList.add('dt-breadcrumb-collapsed-container');
+      this._document.body.appendChild(container);
+      this._collapsedContainer = container;
+    }
+  }
+
+  /**
+   * Returns an array of items that need to be moved to the overlay
+   * @param containerRect Bounds of the breadcrumbs container
+   */
+  private _determineItemsToTransplant(
+    containerRect: DOMRect,
+  ): DtBreadcrumbsItem2[] {
+    return determineOverflowingItems(
+      containerRect,
+      this._itemsWidthMap,
+      COLLAPSED_BUTTON_WIDTH,
+    );
+  }
+
+  /** Returns the first focusable item of the breadcrumbs in the overlay */
+  private _getFirstTransplantedFocusableItem(): DtBreadcrumbsItem2 | null {
+    const focusableTransplantedItems = Array.from(
+      this._itemsPortalsMap.keys(),
+    ).filter((item) => item._isFocusable);
+    return focusableTransplantedItems[0] || null;
+  }
+
+  /** Returns the last focusable item of the breadcrumbs in the overlay */
+  private _getLastTransplantedFocusableItem(): DtBreadcrumbsItem2 | null {
+    const focusableTransplantedItems = Array.from(
+      this._itemsPortalsMap.keys(),
+    ).filter((item) => item._isFocusable);
+    return (
+      focusableTransplantedItems[focusableTransplantedItems.length - 1] || null
+    );
+  }
+
+  /** Returns the first visible breadcrumb that is focusable */
+  private _getFirstVisibleFocusableItem(): DtBreadcrumbsItem2 | null {
+    const visibleItems = Array.from(this._items).slice(
+      this._itemsPortalsMap.size,
+    );
+    const focusableVisibleItems = visibleItems.filter(
+      (item) => item._isFocusable,
+    );
+
+    return focusableVisibleItems[0] || null;
```

Comment:
```suggestion
    return visibleItems.find((item) => item._isFocusable);
```
This should also do the trick and return null if no item could be found.